lib/utils/validateURL/validateURL.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const REGEXP_URL = new RegExp('^$|([Hh][Tt][Tt][Pp]|[Ff][Tt][Pp])([Ss])?://.+$');
+
+export const validateURL = value => {
+  const isValid = REGEXP_URL.test(value);
+
+  if (value === undefined || isValid) return undefined;
```

Comment:
`value === undefined || REGEXP_URL.test(value)`

Reply:
REGEXP_URL.test(value) might return boolean. But outcome is expected as `undefined` in case of valid input